tools/roxvet/analyzers/donotcompareproto/analyzer.go

```diff
@@ -36,26 +36,28 @@ var (
 		"map[string]":   "MapEqual",
 	}
 
-	allowedCallerPackages []string
+	allowedCallerPackages = []string{
+		"github.com/stackrox/rox/central/delegatedregistryconfig/convert",
```

Comment:
What are `allowedCallerPackages` - is that list of packages that this analyzer will ignore. Something like `ignore` for linters? If yes -> can you please document why we are ignoring this package?

Reply:
Fixed them